.github/workflows/ci.yml

```diff
@@ -55,11 +56,19 @@ jobs:
           check_retries: true
           truncate_stack_traces: false
 
+  check-labels:
+    name: "coverage-jacoco-do-not-submit"
+    # Skip coverage test if PR has 'jacoco:skip' label
+    if: ${{ github.event_name != 'pull_request' || !contains(github.event.pull_request.labels, 'jacoco:skip') }}
+    runs-on: ubuntu-latest
+    steps:
+      - run: echo "labels are ${{ toJson(github.event.pull_request.labels) }}"
+
   reporting-jacoco-coverage-failures:
     name: "coverage-jacoco"
-    needs: run-unit-tests
+    # Skip coverage test if PR has 'jacoco:skip' label
```

Comment:
I don't think this really changes anything -  you could read the labels thru github api - but in any case I think it would be better to just stuck with new commits and/or open&close to run this.